python/tests/test_wright_fisher.py

```diff
@@ -38,12 +38,14 @@
 
 class WrightFisherSimulator:
     """
-    SIMPLE simulation of a bisexual, haploid Wright-Fisher population of size N
-    for ngens generations, in which each individual survives with probability
-    survival and only those who die are replaced.  If num_loci is None,
-    the chromosome is 1.0 Morgans long, and the mutation rate is in units of
-    mutations/Morgan/generation. If num_loci not None, a discrete recombination
-    model is used where breakpoints are chosen uniformly from 1 to num_loci - 1.
+    SIMPLE simulation of `num_pops` bisexual, haploid Wright-Fisher populations
```

Comment:
I guess I have to test the new option to do multiple pops/migrations. Any insights into which types of tests are needed?